### PR TITLE
[Bug][CDC] Fix TemporalConversions

### PR DESCRIPTION
**Pull request by hailin0**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/debezium/row/SeaTunnelRowDebeziumDeserializeSchema.java [hunk 1]
```diff
@@ -128,7 +128,7 @@ public void deserialize(SourceRecord record, Collector<SeaTunnelRow> collector)
         } catch (Throwable e) {
             // ignore
         }
-        String tableId = TablePath.of(databaseName, schemaName, tableName).toString();
+        String tableId = TablePath.of(databaseName, schemaName, tableName).toString().toUpperCase();
```
Comment (hailin0) [inline seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/debezium/row/SeaTunnelRowDebeziumDeserializeSchema.java]: SeaTunnelRow#tableId using uppercase

File: seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/debezium/utils/TemporalConversions.java [hunk 2]
```diff
@@ -66,6 +67,9 @@ public static LocalDate toLocalDate(Object obj) {
             return LocalDate.of(date.getYear() + 1900, date.getMonth() + 1, date.getDate());
         }
         if (obj instanceof Long) {
+            if ((Long)obj > ChronoField.EPOCH_DAY.range().getMaximum()) {
```
Comment (hailin0) [inline seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/debezium/utils/TemporalConversions.java]: fix millisecond

**Comment by hailin0** [comment]
image

File: seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/debezium/row/SeaTunnelRowDebeziumDeserializeSchema.java (hunk 1 above)
Comment (Hisoka-X) [reply seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/debezium/row/SeaTunnelRowDebeziumDeserializeSchema.java]: If tableName case sensitive, there are a little problem.

File: seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/oracle/OracleTypeMapper.java [hunk 3]
```diff
@@ -68,15 +68,20 @@ public class OracleTypeMapper implements JdbcDialectTypeMapper {
...
     @Override
     public SeaTunnelDataType<?> mapping(ResultSetMetaData metadata, int colIndex)
             throws SQLException {
-        String oracleType = metadata.getColumnTypeName(colIndex).toUpperCase();
-        String columnName = metadata.getColumnName(colIndex);
-        int precision = metadata.getPrecision(colIndex);
-        int scale = metadata.getScale(colIndex);
-        switch (oracleType) {
+        return mapping(
+                metadata.getColumnName(colIndex),
```
Comment (Hisoka-X) [inline seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/oracle/OracleTypeMapper.java]: Why remove `toUpperCase`?

**Review by Hisoka-X — APPROVED** [review]

**Review by TyrantLucifer — APPROVED** [review]

**Pull request merged**